bin/common.bash

```diff
@@ -228,3 +228,21 @@ check_openstack_credentials() {
         exit 1
     fi
 }
+
+# Checks what difference between the main repo and the sun repo and open a dialog if they differs.
+diff_check(){
+pushd "${here}/../" || exit
+
+result=$(git diff kubespray/)
```

Comment:
It would be really cool if we also fetch the current commit that the repo is at?
Then we could present that as info like:

```text
Kubespray commit mismatch.
Expected  "${commit/tag that it should be}", got "${currently checked out commit/tag}" 
Do you want to continue (y/n):